src/gettingstarted/own-code/routes-configuration.md

```diff
@@ -0,0 +1,57 @@
+# Import your own code
+
+## Configure routes
+
+The final configuration file you will need to modify in your repository is the `.platform/routes.yaml` file, which describes how an incoming HTTP request is going to be processed by Platform.sh.
+
+```.
+├── .platform
+│   ├── routes.yaml
+│   └── services.yaml
+├── .platform.app.yaml
+└── < application code >
+```
+
+A minimal configuration `.platform/routes.yaml` for all languages will look very similar:
+
+{% codetabs name="PHP", type="php", url="https://raw.githubusercontent.com/platformsh/template-php/master/.platform/routes.yaml" -%}
+
+{%- language name="Go", type="js", url="https://raw.githubusercontent.com/platformsh/template-golang/master/.platform/routes.yaml" -%}
+
+{%- language name="Python", type="py", url="https://raw.githubusercontent.com/platformsh/template-python3/master/.platform/routes.yaml" -%}
+
+{% language name="Node.js", type="js", url="https://raw.githubusercontent.com/platformsh/platformsh-example-nodejs/master/.platform/routes.yaml" -%}
+
+{%- endcodetabs %}
```

Comment:
As with services, there's no reason this needs to vary by language.  Just inline a single example.

Reply:
Got it. PHP seems to be the best commented version, so I'm using that. As for services, I've updated it to the language-examples services.yaml. Don't know if that's too long though...